sdk/python/feast/feature_store.py

```diff
@@ -1869,39 +1869,32 @@ def retrieve_online_documents(
             allow_cache=True,
             hide_dummy_entity=False,
         )
-        if features:
```

Comment:
this logic was to allow for users to avoid having to have both `feature` and `features`.